src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/CaseAccessMetadata.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
+
+import lombok.Data;
+
+@Data
+public class CaseAccessMetadata {
+
+    public static final String ACCESS_GRANTED = "[ACCESS_GRANTED]";
+    public static final String ACCESS_PROCESS = "[ACCESS_PROCESS]";
+
+    public static final String ACCESS_GRANTED_LABEL = "Access Granted";
+
+    private String accessGrants;
```

Comment:
Can we use enums as they are available already? 
how about keep the property as `private List<GrantType> accessGrants` and expose a additional method for the comma-separated string String getAccessGrantsString() ?